### PR TITLE
Add syntactic sugar for getting the current Fiscal{Year,Quarter}.

### LINKED ISSUE #2: Syntactic sugar for getting the current Fiscal{Year,Quarter}
I think it would be nice if there was an easy way to get the current `FiscalYear`/`FiscalQuarter`.

This could easily be achieved by using a couple of `classmethods`. E.g.:

``` diff
$ git diff
diff --git a/fiscalyear.py b/fiscalyear.py
index c6f8ddc..f7213d7 100644
--- a/fiscalyear.py
+++ b/fiscalyear.py
@@ -191,6 +191,11 @@ class FiscalYear(object):
         self._fiscal_year = fiscal_year
         return self
 
+    @classmethod
+    def current(cls):
+        today = datetime.date.today()
+        return cls(today.year)
+
     def __repr__(self):
         """Convert to formal string, for repr().
 
@@ -371,6 +376,11 @@ class FiscalQuarter(object):
         self._quarter = quarter
         return self
 
+    @classmethod
+    def current(cls):
+        today = datetime.date.today()
+        return cls(today.year, (today.month // 3) + 1)
+
     def __repr__(self):
         """Convert to formal string, for repr().
```

This way you could write: 
``` python
In [3]: FiscalYear.current()
Out[3]: FiscalYear(2018)

In [4]: FiscalQuarter.current()
Out[4]: FiscalQuarter(2018, 1)
```

If you want, I can try to make a pull request.

### PR DESCRIPTION
**Pull request by pmav99**
Fixes #2 

I had to add `pytest-mock` in the dependencies.

I also thought that it would be better if a native speaker wrote the docs.

Anyway, check it out and let me know of any remarks.

### CODE REVIEW

**Comment by adamjstewart** [comment]
Thanks! I'll try to get a chance to review this and your other PR sometime in the next week. I have exams coming up (student life...) so I apologize if I get side-tracked. Ping me if I don't respond in a week.

**Comment by pmav99** [comment]
ping!

File: README.rst [hunk 1]
```diff
@@ -61,6 +68,13 @@ These objects represent the standalone ``FiscalQuarter`` class.
    >>> b in a
    True
 
+You can also get the current ``FiscalQuarter`` with:
+
+.. code-block:: python
+
+   >>> from fiscalyear import *
+   >>> FiscalQuarter.current()
+   FiscalQuarter(2018, 2)
```
Comment (adamjstewart) [inline README.rst]: Can you also add these same snippets to `docs/basic_usage.rst` in the same locations? Regrettably these files are almost duplicates of each other. I haven't found a good way to include one from the other yet. GitHub doesn't support rST includes 😢

File: fiscalyear.py [hunk 2]
```diff
@@ -191,6 +191,16 @@ def __new__(cls, fiscal_year):
         self._fiscal_year = fiscal_year
         return self
 
+    @classmethod
+    def current(cls):
+        """ Alternartive constructor. Returns the current FiscalYear.
```
Comment (adamjstewart) [inline fiscalyear.py]: Typo: "Alternartive" -> "Alternative"

Also, can you remove the leading space?

File: fiscalyear.py [hunk 3]
```diff
@@ -371,6 +381,16 @@ def __new__(cls, fiscal_year, quarter):
         self._quarter = quarter
         return self
 
+    @classmethod
+    def current(cls):
+        """ Alternartive constructor. Returns the current FiscalQuarter.
```
Comment (adamjstewart) [inline fiscalyear.py]: Same comment as above.

File: requirements.txt [hunk 4]
```diff
@@ -1,6 +1,7 @@
 codecov
 pytest
 pytest-cov
+pytest-mock
```
Comment (adamjstewart) [inline requirements.txt]: I'm not opposed to adding new dependencies, but can modify the first paragraph in `docs/testing.rst`? Something like:
```rst
To run the test-suite, you will need to install the `pytest <https://docs.pytest.org/en/latest/>`_ and `pytest-mock <https://pypi.python.org/pypi/pytest-mock>`_ packages.
```

File: README.rst [hunk 5]
```diff
@@ -33,6 +33,13 @@ The ``FiscalYear`` class provides an object for storing information about the st
    >>> a.end
    FiscalDateTime(2017, 9, 30, 23, 59, 59)
 
+You can also get the current ``FiscalYear`` with:
+
+.. code-block:: python
+
+   >>> from fiscalyear import *
```
Comment (adamjstewart) [inline README.rst]: I would drop this import line here and below. After the first code block I assume everyone will have fiscalyear imported.

File: docs/testing.rst [hunk 6]
```diff
@@ -1,8 +1,13 @@
...
 =======
 
-``fiscalyear`` comes with a full test-suite called ``test_fiscalyear``. To run the test-suite, you will need to install the `pytest <https://docs.pytest.org/en/latest/>`_ package.
+``fiscalyear`` comes with a full test-suite called ``test_fiscalyear``. To run the test-suite, you
+will need to install the following packages:
 
+* `pytest <https://docs.pytest.org/en/latest/>`_
+* `pytest-cov <https://docs.pytest.org/en/latest/>`_
+* `pytest-mock <https://docs.pytest.org/en/latest/>`_
+* `pytest-runner <https://docs.pytest.org/en/latest/>`_
```
Comment (adamjstewart) [inline docs/testing.rst]: Technically `pytest-cov` and `pytest-runner` aren't required to run the unit tests, only to run them with `setup.py` or with coverage analysis. Not sure if we should leave those out and mention them elsewhere or not.

**Comment by adamjstewart** [comment]
Ping @pmav99. Aside from my last 2 requested changes, I think this PR is ready to go!

File: docs/testing.rst (hunk 6 above)
Comment (pmav99) [reply docs/testing.rst]: I think that they can be considered dev dependencies and that it is OK to mention them. Making distinctions between mandatory and optional dependencies feels out of scope of this request.

File: docs/basic_usage.rst [hunk 7]
```diff
@@ -17,6 +17,14 @@ The ``FiscalYear`` class provides an object for storing information about the st
    >>> a.end
    FiscalDateTime(2017, 9, 30, 23, 59, 59)
 
+You can also get the current ``FiscalYear`` with:
+
+.. code-block:: python
+
+   >>> from fiscalyear import *
```
Comment (adamjstewart) [inline docs/basic_usage.rst]: Remove import here too

File: README.rst [hunk 8]
```diff
@@ -26,13 +26,18 @@ The ``FiscalYear`` class provides an object for storing information about the st
 
 .. code-block:: python
 
-   >>> from fiscalyear import *
```
Comment (adamjstewart) [inline README.rst]: You should add this import back

**Comment by adamjstewart** [comment]
Thanks for the PR! I'll play around with the docs and do some testing and try to put out a new release by the end of the week. Sorry it took so long to get this merged!

**Pull request merged**